WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt

```diff
@@ -632,7 +632,7 @@ d_jewish = {
 	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
 }
 
-d_melchisedechianism = {
+k_melchisedechianism = {
```

Comment:
It is called the _**KINGDOM**_ of Salem. It is a SECULAR title (like the Caliphates) that is, and always has been, a _**KINGDOM**_. If you are really going to dispute me on giving the _**KINGDOM**_ of Salem its actual tier, you need to rethink your animosity towards me.